pkg/transforms/conversion.go

```diff
@@ -64,7 +64,7 @@ func (f Conversion) TransformToJSON(edgexcontext *appcontext.Context, params ...
 	if result, ok := params[0].(dtos.Event); ok {
 		b, err := json.Marshal(result)
 		if err != nil {
-			// LoggingClient.Error(fmt.Sprintf("Error parsing JSON. Error: %s", err.Error()))
+			// lc.Error(fmt.Sprintf("Error parsing JSON. Error: %s", err.Error()))
```

Comment:
remove dead code?

Reply:
Yup, looks like it. My rename picked it up in the comment...